client/components/DataManagement/index.jsx

```diff
@@ -94,143 +115,154 @@ const DataManagement = () => {
+        <h1>Data Management</h1>
+
+        {emptyTestPlans && (
+          <h2 data-testid="data-management-no-test-plans-h2">
+            There are no Test Plans available
+          </h2>
+        )}
+
+        {emptyTestPlans && isAdmin && (
+          <Alert
+            key="alert-configure"
+            variant="danger"
+            data-testid="data-management-no-test-plans-p"
+          >
+            Add a Test Plan to the Queue
+          </Alert>
+        )}
+
+        {isAdmin ? (
+          <>
+            <h2>Introduction</h2>
+            <p data-testid="data-management-instructions">
+              This page provides a view of the latest test plan version
+              information, and where they currently are in the{' '}
+              <a href="https://github.com/w3c/aria-at/wiki/Working-Mode">
+                ARIA-AT Community Group’s review process
+              </a>
+              .<br />
+              Use this page to manage Test Plans in the Test Queue and their
+              phases.
+            </p>
+
+            <ManageTestQueue
+              ats={ats}
+              testPlanVersions={testPlanVersions}
+              triggerUpdate={refetch}
             />
-            <th>R&D Version</th>
-            <th>Draft Review</th>
-            <th>Candidate Review</th>
-            <th>Recommended Version</th>
-          </tr>
-        </thead>
-        <tbody>
-          {sortedTestPlans.map((testPlan, index) => {
-            return (
-              <DataManagementRow
-                key={testPlan.id}
-                isAdmin={isAdmin}
-                ats={ats}
-                testPlan={testPlan}
-                testPlanVersions={[
-                  ...deprecatedTestPlanVersions,
-                  ...testPlanVersions
-                ].filter(
-                  testPlanVersion =>
-                    testPlanVersion.testPlan.directory === testPlan.directory
-                )}
-                tableRowIndex={index}
-                setTestPlanVersions={setTestPlanVersions}
+          </>
+        ) : (
+          <>
+            <h2>Introduction</h2>
+            <p data-testid="data-management-instructions">
+              This page provides a view of the latest test plan version
+              information, and where they currently are in the{' '}
+              <a href="https://github.com/w3c/aria-at/wiki/Working-Mode">
+                ARIA-AT Community Group’s review process
+              </a>
+              .
+            </p>
+          </>
+        )}
+
+        <h2>Test Plans Status Summary</h2>
+        <FilterButtons
+          filterLabel="Filter"
+          filterAriaLabel="Filter tests plans by status"
+          filterOptions={filterLabels}
+          activeFilter={filter}
+          onFilterChange={setFilter}
+        />
+
+        {isAdmin && (
+          <Button variant="primary" onClick={handleImportTests}>
+            Import Latest Test Plan Versions
+          </Button>
+        )}
```

Comment:
I prefer only having this on the /settings page. If you disagree then perhaps the button can be somewhere else? I feel it is confusing to have it between the filter buttons and the table that they filter.

Reply:
I'm not a big fan of it here either. I tried to plug it in mainly because of the original request being to have the button on the Data Management page but I feel it more fitting by itself on the Settings page. So removing here isn't an issue to me!